src/utils/parse-arguments-into-options.ts

```diff
@@ -73,7 +83,17 @@ export async function parseArgumentsIntoOptions(rawArgs: Args): Promise<RawOptio
     project,
     install: hasInstallRelatedFlag ? install || !skipInstall : null,
     dev,
-    extensions: null, // TODO add extensions flags
     externalExtension: extension,
+    solidityFramework,
   };
 }
+
+function solidityFrameworkHandler(value: string) {
+  const lowercasedValue = value.toLowerCase();
+  if (lowercasedValue === "hardhat" || lowercasedValue === "foundry" || lowercasedValue === "none") {
+    return lowercasedValue;
+  }
```

Comment:
I'm wondering if we could have these exported from somewhere, to avoid hardcoding the value. I've seen it in other places (PRs) too. And also here we could do something like:

`solidityFrameworks.includes(lowercasedValue)`

My first instinct was to look in config.ts file, but It might be tricky because I see `extensions: ["hardhat", "foundry", null]`. We are doing an extra _step_ with the null / none thing in `nullExtensionChoice` (prompt-for-missing-options.ts).

Any ideas, Rinat?

It's fine if we don't fix it in this PR, just thinking out loud!

Reply:
Updated condition but moved array just out of the function, since it's used only here for now.

Regarding `null`/`none`. `none` is used when the user explicitly wants to say that he doesn't need solidity framework `yarn cli -f none`. `null` means that `--solidity-framework/-f` option was not used and cli will ask about it. So looks like it's fine.

I will try to tinker with it later and probably find a better solution

Reply:
> I've seen it in other places (PRs) too.

Ohh yes, actually I have used it in #55, I will try to hoist this logic at some common place in that PR 🙌